shared/lib/ui-utils.js

```diff
@@ -11,3 +11,9 @@ export const ETHERSCAN_PRIVACY_LINK = 'https://etherscan.io/privacyPolicy';
 export const CONSENSYS_PRIVACY_LINK = 'https://consensys.net/privacy-policy/';
 export const AUTO_DETECT_TOKEN_LEARN_MORE_LINK =
   'https://consensys.net/privacy-policy/';
+
+///: BEGIN:ONLY_INCLUDE_IN(blockaid)
+export const BLOCKAID_TERMS_OF_USE =
+  'http://blockiad.io/legal/metamask-ppom-privacy-policy/';
```

Comment:
There is a typo in `blockiad` so the URL is failing (see 1st pic).
However, if I fix it to `blockaid` I also see an error page (see 2nd pic). We should double-check this link

![Screenshot from 2023-10-25 10-07-24](https://github.com/MetaMask/metamask-extension/assets/54408225/2ec3fca2-8e0a-483c-a8a3-8a440b7b3387)

![Screenshot from 2023-10-25 10-06-23](https://github.com/MetaMask/metamask-extension/assets/54408225/71eba364-99ce-4851-b288-9423b1326eb8)

Reply:
I'll fix the typo, but @bschorchit mentioned the blockaid team is supposed to fix the page not found issue

Reply:
thank you @segun , about the `http`, could we `https` instead? (if @bschorchit / Blockaid agrees)

Reply:
Waiting for blockaid team to fix the link before merging this.